src/asgen/zarchive.d

```diff
@@ -171,16 +180,23 @@ private:
 
     archive *openArchive ()
     {
+        import core.stdc.string : strerror;
```

Comment:
You could move that import into the `if (ret != ARCHIVE_OK) {` block, for less visibility of "strerror", or, since it's used in multiple functions and has a fairly unique name, just move it to the toplevel imports and have less import statements.

Reply:
Done, thanks!